src/pcapi/core/offerers/validation.py

```diff
@@ -41,6 +41,11 @@ def check_venue_edition(modifications, venue):
         errors = ApiErrors()
         errors.add_error("siret", "Vous ne pouvez pas modifier le siret d'un lieu")
         raise errors
+    if siret:
+        venue_with_same_siret = Venue.query.filter(Venue.siret == siret).first()
```

Comment:
(détail / style)

```suggestion
        venue_with_same_siret = Venue.query.filter_by(siret=siret).one_or_none()
```

J'aime bien utiliser `filter_by()` quand c'est possible, l'argument est plus concis. (Question de goût, peut-être !)

C'est équivalent, mais ça clarifie le fait que le SIRET est unique en base. (Et je trouve `one_or_none()` un peu plus dans ce cas, mais là aussi c'est peut-être subjectif !)